smalltalksrc/Slang/TVariableNode.class.st

```diff
@@ -127,7 +127,11 @@ TVariableNode >> printOn: aStream level: level [
 { #category : #accessing }
 TVariableNode >> setName: aString [
 
-	name := aString.
+	(CCodeGenerator reservedWords includes:
+		 aString) ifTrue: [
+		TranslationError signal: aString , ' is a reserved word' ].
```

Comment:
For me, this is _polémico_. Do we want this validation in the setter? I was thinking to have this validation before the next transformation. But I don't know how it is used...

Also, we can have a `validateReservedWords` method as in `TMethod`.